changelog.txt

```diff
@@ -31,6 +31,7 @@
 * Fix - Add protection around func_get_args_call for backwards compatibility. #28677
 * Fix - Restore stock_status in REST API V3 response. #28731
 * Fix - Revert some of the changes related to perf enhancements (27735) as it was breaking stock_status filter. #28745
+* Fix - Reverts #28204 to ensure compatibility with extensions using legacy do_action calls. #28835
```

Comment:
Shouldn't the changelog have a new entry for 4.9.1 instead of adding the item to 4.9.0?